rllib/evaluation/collectors/simple_list_collector.py

```diff
@@ -133,7 +140,7 @@ def build(self, view_requirements: Dict[str, ViewRequirement]) -> \
                 continue
             # OBS are already shifted by -1 (the initial obs starts one ts
             # before all other data columns).
-            shift = view_req.shift - \
+            shift = view_req.data_rel_pos - \
```

Comment:
Renamed this b/c this will support (in the upcoming PRs) not just a single shift (int), but also:
- list of ints (include not just one ts in this view, but several)
- a range string, e.g. "-50:-1" (will be used by attention nets and Atari framestacking).